build/common.sh

```diff
@@ -203,6 +207,18 @@ function kube::build::is_osx() {
   [[ "$(uname)" == "Darwin" ]]
 }
 
+function kube::build::update_dockerfile() {
+  if kube::build::is_osx; then
```

Comment:
@ixdy I saw this suggestion from you before I made the fix.
On GNU linux it works, if we echo the command we can see the command is changed to sed -i ""; however on BSD ${sed_opts[@]} will drop the "", if we echo the command, it prints sed -i xxxx, I do not get a way to print the "" as array element, that's why I changed it to current solution.